### DISCUSSION TITLE
Rozszerzenie działa, jednak nie działa opóźnienie

### DISCUSSION THREAD

**Discussion by JacekU** [Endiasz/Instaling-Bot#6] [Ideas]
Nie działa mi w ogóle ustawienie opóźnienia pomiędzy kolejnymi wpisywaniami (bardzo szybko zapierdziela strona po stronie). By instaling miał trudniej wykryć kombinacje dodałem do twojego script.js:

function getRandomInt(max) {
  var res = 10;
  while (res<4000)
    res=Math.floor(Math.random() * max);
  return res;
}

var timeBetween2 = 0;
	timeBetween2 = getRandomInt(6000);
var TheLoopInterval = setInterval(theLoopFunction(), timeBetween2);

Czy możesz zmodyfikować rozszerzenie by tak działało (brało do 6 sekund przerwy między kolejnymi słowami lub poprawić by działał odstęp wpisany w konfiguracji?

**Comment by Endiasz** [comment]
Siema zobaczę czy dzisiaj uda mi się naprawić problem, jeżeli nie to skorzystam z twojego rozwiązania. Dzięki za informacje

**Reply by JacekU** [reply]
Dzięki, potrzebuje tego dla dzieci, instaling to piece of shit

**Reply by Endiasz** [reply]
tak na marginesię nie sądzę że instaling to wykrywa bo sam parę razy razy to u siebie puszczałem i nic nie było nikt też się nie skarżył.

**Reply by Endiasz** [reply]
Wrzuconie, pobież sobie od nowa, Aktualna wersja to 4.9 co prawda nie daje losowych czasów ale można sobie dostosować czas między powtórzeniami

**Reply by JacekU** [reply]
No mistrzuniu, to działa!!! wrzuć na chrome

**Reply by Endiasz** [reply]
Wiesz co, chciałem to zrobić, nawet nie tak dawno próbowałem ale tam jakieś problemy z weryfikacją miałem. Będę próbował i najwyżej wrzucę linka
